core/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -32,23 +32,17 @@ private[spark] trait Logging {
 
   // Make the log field transient so that objects with Logging can
   // be serialized and used on another machine
-  @transient private var log_ : Logger = null
+  @transient lazy val log : Logger = {
```

Comment:
Yup, my bad. Fixed via
https://github.com/infynyxx/spark/commit/18ae8c5b0184bed17e224963b1d90e709cb8faa4

On Wed, Jun 22, 2016 at 3:15 PM, Sean Owen <notifications@github.com> wrote:

> In core/src/main/scala/org/apache/spark/internal/Logging.scala
> <https://github.com/apache/spark/pull/13842#discussion_r68072052>:
>
> > @@ -32,23 +32,17 @@ private[spark] trait Logging {
> >
> >    // Make the log field transient so that objects with Logging can
> >    // be serialized and used on another machine
> > -  @transient private var log_ : Logger = null
> > +  @transient lazy val log : Logger = {
>
> I think this will fail style checks because of the space before the colon
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/spark/pull/13842/files/986e61f96ece6d7f71ff760384cc7553fe3828f4#r68072052>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACdKtAjTKSCyAI2oyyH7fFUatd4xsCmks5qOVGUgaJpZM4I7vOl>
> .
>



-- 
--
Cheers,
Praj